x/bridge/types/params.go

```diff
@@ -0,0 +1,117 @@
+package types
+
+import (
+	"fmt"
+
+	errorsmod "cosmossdk.io/errors"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	paramtypes "github.com/cosmos/cosmos-sdk/x/params/types"
+
+	"github.com/osmosis-labs/osmosis/osmoutils"
+)
+
+var (
+	KeySigners = []byte("Signers")
+	KeyAssets  = []byte("Assets")
+)
+
+func NewParams(signers []string, assets []AssetWithStatus) Params {
+	return Params{
+		Signers: signers,
+		Assets:  assets,
+	}
+}
+
+// DefaultParams creates default x/bridge params.
+func DefaultParams() Params {
+	return Params{
+		Signers: []string{},
+		Assets:  DefaultAssetsWithStatuses(),
+	}
+}
+
+// Validate x/bridge params.
+func (p Params) Validate() error {
+	if len(p.Signers) == 0 {
+		return errorsmod.Wrapf(ErrInvalidSigners, "Signers are empty")
+	}
```

Comment:
I think this should actually be a valid state right? There might be a world in which we want no signers (this is the default state as well)

Reply:
Yes, it makes sense. Moreover, now the module won't even start since the default genesis can't pass the validation :)
Deleted this check.